ui/perfherder/alerts/StatusDropdown.jsx

```diff
@@ -257,38 +259,78 @@ export default class StatusDropdown extends React.Component {
     alertStatus === 'investigating' ||
     (alertStatus !== status && this.isResolved(alertStatus));
 
+  getDifference(date1, date2) {
```

Comment:
Suggestion: Rename date1 and date2 to first_date and second_date? Or something similar instead of using 1 and 2.